src/st/e_skeleton.h

```diff
@@ -72,6 +86,18 @@ void EntitySkeleton(Entity* self) {
         self->ext.skeleton.attackTimer = 80; // Skeleton attack timer cycle
         self->ext.skeleton.facingLeft = 0;   // Facing init
         self->ext.skeleton.attackTimerIndex = 0;
+#ifdef STAGE_IS_LIB
+        if (self->params & 0x10) {
+            self->flags |= FLAG_UNK_2000;
+            self->hitboxState = 0;
+            self->animCurFrame = 0;
+            self->step = SKELETON_LIB_UNK;
+            return;
+        }
+        break;
+    case SKELETON_LIB_UNK:
+        func_us_801BB8DC(sensors_ground);
```

Comment:
Not sure what this function is just yet but it's used in a few places in LIB, looks like a collision function.